data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/network_action/PstSetpoint.java

```diff
@@ -47,11 +59,14 @@ public void setSetpoint(double setpoint) {
     /**
      * Change tap position of the PST pointed by the network element at the tap given at object instantiation.
      *
-     * @param network: network to modify
+     * @param network network to modify
      */
     @Override
     public void apply(Network network) {
-        PstRange pst = new PstRange(networkElement);
+        AbsoluteFixedRange range = new AbsoluteFixedRange(Double.NEGATIVE_INFINITY, Double.POSITIVE_INFINITY, RangeDefinition.CENTERED_ON_ZERO);
+        List<AbstractRange> ranges = new ArrayList<>();
+        ranges.add(range);
+        PstRange pst = new PstRange(getId(), getName(), getOperator(), getUsageRules(), ranges, networkElement);
```

Comment:
here a simple constructor with only an id and a network element would be useful

Reply:
done